lib/delayed/priority.rb

```diff
@@ -82,6 +82,15 @@ def alerts=(alerts)
         @alerts = alerts&.sort_by { |k, _| names.keys.index(k) }&.to_h
       end
 
+      def assign_at_midpoint?
+        @assign_at_midpoint || false
+      end
+
+      def assign_at_midpoint=(assign_at_midpoint=false)
```

Comment:
```suggestion
      def assign_at_midpoint=(assign_at_midpoint)
```

Argument defaults don't work on assignment methods.

Reply:
Yeah, that makes sense. You probably can't write `thing.attribute=` anyway